client/src/periodUtils.js

```diff
@@ -163,10 +163,12 @@ export const getPeriodsConfig = (
   const periods = []
   for (let i = numberOfPeriods - 1; i >= 0; i--) {
     const periodDetails = PERIOD_FACTORIES[recurrence](now, offset + i)
+
     if (forAssessments) {
-      // only allow assessments for past periods
-      periodDetails.allowNewAssessments = offset + i > 0
+      // allow new assessments
+      periodDetails.allowNewAssessments = true
```

Comment:
In the context of periodic assessments, this should at least check the assessment configuration to see if future assessments are allowed.